CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ Usage:
 * Fix #2224: Quarkus native base image read from properties (configurable)
 * Fix #2228: Quarkus native base image uses UBI 8.7
 * Fix #2290: JKube is not picking docker credentials from `~/.docker/config.json` file
+* Fix #2293: JibServiceUtil pushes separate images for additional tags specified in build configuration
```

Comment:
I think you have been mixing 2 different PRs here

Reply:
I didn't add this line... I think it's from a merge/rebase...